.github/workflows/docker_builds.yml

```
@@ -168,6 +168,29 @@ jobs:
         tag_names: true
         workdir: docker
 
+  px4-dev-nuttx-jammy:
```

```suggestion
px4-dev-base-jammy:
    name: "Ubuntu Jammy Base"
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v3
    - name: Build container
      if: github.event_name == 'pull_request'
      run: |
        cd docker
        make px4-dev-base-jammy
    - name: Build & Publish container
      if: github.event_name == 'push' && (contains(github.ref, '/heads/master') || contains(github.ref, '/tags/'))
      uses: julianoes/Publish-Docker-Github-Action@master
      with:
        name: px4io/px4-dev-base-jammy
        username: ${{ secrets.DOCKER_USERNAME }}
        password: ${{ secrets.DOCKER_PASSWORD }}
        dockerfile: Dockerfile_base-jammy
        cache: false
        tag_names: true
        workdir: docker

  px4-dev-nuttx-jammy:
```
no Jammy Base build?

---

my mistake, i added one now